Sources/TensorFlow/Operators/Math.swift

```diff
@@ -619,7 +619,7 @@ internal func _vjpTan<T: TensorFlowFloatingPoint>(
     return (value, { v in v * (1 + value.squared()) })
 }
 
-/// Computes `sinh` of the specified tensor element-wise.
+/// Returns `sinh` of the specified tensor element-wise.
```

Comment:
@rxwei what do you think about 
```suggestion
/// Returns the hyperbolic sine of the specified tensor element-wise.
```
and similarly for all hyperbolic functions

Reply:
I see what your saying! Sounds good. I'll go through those and give a bit more explanation where possible.